Source/Core/DiscIO/WIABlob.h

```diff
@@ -56,6 +56,7 @@ class WIARVZFileReader : public BlobReader
   u64 GetBlockSize() const override { return Common::swap32(m_header_2.chunk_size); }
   bool HasFastRandomAccessInBlock() const override { return false; }
   std::string GetCompressionMethod() const override;
+  u64 GetCompressionLevel() const override { return Common::swap32(m_header_2.compression_level); }
```

Comment:
This works incorrectly for negative compression levels, which are supported by zstd. (Note that `compression_level` should be interpreted as unsigned for WIA but signed for RVZ. Not that I think anyone would notice if we used signed for WIA...)